### PR TITLE
Promote support for replica mv in resource_bigquery_table to GA

### PR DESCRIPTION
**Pull request by wj-chen**
Promote support for replica mv in resource_bigquery_table to GA. This is a follow-up to https://github.com/GoogleCloudPlatform/magic-modules/pull/9773.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
bigquery: promoted support for replica mv in `resource_bigquery_table` to GA
```

### CODE REVIEW

**Comment by c2thorn** [comment]
going to run GA tests in our env

**Review by c2thorn — APPROVED** [review]
tests passed, LGTM on my end. will wait for the other approval.

**Review by sachinpro — APPROVED** [review]
LGTM!

**Pull request merged**
